src/context/analysisResultStateSlice.ts

```diff
@@ -101,10 +99,6 @@ function getAdminBoundariesURL() {
   if (adminBoundariesPath.startsWith('http')) {
     return adminBoundariesPath;
   }
-  // do not send a local path to the API, use a fixed boundary file instead.
-  if (isLocalhost) {
```

Comment:
this will break when the API is being run locally, no?

Reply:
No, all works fine